### ISSUE TITLE
Is the introduction still fit for purpose?

### ISSUE THREAD

**Issue by ElenaFdR** [publiccodenet/standard#425]
Does it still work for new readers? Is it still an accurate reflection of our aspirations for the Standard? And more importantly, does it reflect everything we've learned about how people (want to) use the Standard in the intervening 1.5+ years since it was written?

People have already raised separate issues about specific parts of the introduction:

- whether the Foundation's stewardship process needs such a detailed explanation [#396]
- making the introduction more relevant to the audiences addressed in the rest of the Standard [#280]
- adding a clearer values based entry point to the Standard [#318]
- ambiguous support for commercial licence models [#185]
- and most damningly, bad writing [#216]

One additional logistical reason to update it now is from the discussions on translating the Standard. We've discussed encouraging widespread translation of the introduction, but leaving the criteria in English (so that there's no ambiguity in the functional requirements). If widespread translation of the introduction is to have any of the impact we hope, then the intro needs to be the strongest, tightest, most compelling version of itself that it can be.

**Comment by Ainali** [comment]
In the last community call we discussed this issue and here is [the blog post](https://blog.publiccode.net/community%20call/2021/01/25/notes-from-community-call-7-january-2021.html) that summarized the call.

TL;DR We will start by restructuring the introduction and moving around a few paragraphs.

**Comment by ericherman** [comment]
Improving the introduction to match the more current landscape means changing the tone from "call-to-arms of a new world" to one of supporting policy goals of current digital transformations.

**Comment by ericherman** [comment]
The Foundation's stewardship process is located at https://publiccode.net/standard-for-public-code/ and not duplicated in the standard.

By transforming the Introduction into a Foreword, this removes the dissonance in the tone from what makes sense in the text of a standard.

There is still room for transforming the reader's guide to an actual introduction relevant to reader's of the standard, but perhaps that should be its own issue.

A value-based foreword would certainly be an improvement, that should be a part of https://github.com/publiccodenet/standard/issues/216 I think.

**Comment by ericherman** [comment]
I have a proposal for how I would like to hack on this, ideally co-working in one of our [community calls](https://community.standardforpubliccode.org/):

1. brain-storm elements we think a foreword should contain (any order)
2. review the Foundation foreword to see if there are any elements missed.
3. create a rough outline
4. transform outline to real text, aiming for a voice that better reflects the current community: more "sober" and "matter of fact" not a "call to arms" with "we believe" statements
